packages/ra-ui-materialui/src/input/AutocompleteInput.js

```diff
@@ -281,22 +299,13 @@ export class AutocompleteInput extends React.Component {
                         isRequired={isRequired}
                     />
                 }
-                value={value}
-                onChange={onChange}
-                autoFocus={autoFocus}
-                margin="normal"
-                className={classnames(classes.root, className)}
-                inputRef={ref}
                 error={!!(touched && error)}
                 helperText={(touched && error) || helperText}
+                className={classnames(classes.root, className)}
                 {...options}
-                InputProps={{
-                    classes: {
-                        input: classes.input,
-                    },
-                    ...InputProps,
-                    ...other,
-                }}
+                {...input}
```

Comment:
why did you add these props?

Reply:
I brought the component inline with `TextInput`, which is better designed. Same applies for this component.